audeer/core/utils.py

```diff
@@ -204,7 +204,7 @@ def run_worker_threads(
         num_workers: int = None,
         progress_bar: bool = False,
         task_description: str = None
-) -> Sequence[Any]:
+) -> Sequence[Any]:  # pragma: nocover
```

Comment:
I'm always using `pragma: no cover`, but from the test coverage we see that both works :)

Reply:
I did so before.
Changed it here and at another place.